google/cloud/storage/internal/curl_client.cc

```diff
@@ -104,8 +104,22 @@ bool XmlEnabled() {
 
 }  // namespace
 
+std::string HostHeader(Options const& options, char const* service) {
+  // This function returns an empty string if In most cases libcurl will fill
+  // out the correct `Host: ` header based on the URL.  The main exception is
+  // `https://restricted.googleapis.com` Hard-code the Host header. Applications
```

Comment:
s/Hard-code/where we hard-code/ perhaps?

Reply:
See above